mp_cnn/evaluation.py

```diff
@@ -2,7 +2,8 @@
 from mp_cnn.evaluators.msrvid_evaluator import MSRVIDEvaluator
 from mp_cnn.evaluators.trecqa_evaluator import TRECQAEvaluator
 from mp_cnn.evaluators.wikiqa_evaluator import WikiQAEvaluator
-
+from nce.NCE_MP_Pytorch.evaluators.trecqa_evaluator import TRECQAEvaluator_NCE
```

Comment:
There's some repetition in the module path.

`from nce.mp_cnn.evaluators.trecqa_evaluator` seems better to me, module path should not have capital letters and we already know it is PyTorch so that is redundant.

Please do this for all imports.

Reply:
NCE_MP_Pytorch is a folder. I need to change them to the smaller case. So it means folder name is supposed to be the smaller case, right?

Reply:
yep change folder name to all lower case, and do not repeat things already in the path like nce if this is already in nce parent folder and do not add pytorch anywhere since it's obvious.